app/views/shared/fields/_note.html.erb

```diff
@@ -46,7 +46,7 @@
         elsif note_type != nil && !note_type.blank?
           notes[note_type.titleize] = note
         else
-          notes['General Note'] = note
+          notes['General Note'.to_s] = note
```

Comment:
Thanks for explaining. I think that while creating a new String, and thus a new object ID, does work, it is quite cryptic.

Appending the values to each note array is going to be more intuitive for our team to read in the future if other changes are needed. It makes perfect sense in many cases that we end up with an array of General Notes, and potentially other notes as well.